tests/integration/clientv3/naming/resolver_test.go

```diff
@@ -110,7 +110,6 @@ func TestEtcdGrpcResolver(t *testing.T) {
 			}
 			t.Fatalf("unexpected response from foo (e2): %s", resp.GetPayload().GetBody())
 		}
-
```

Comment:
FYI: 

git commit -amend 
git push --force

Is usual "hack" to re-trigger tests.

Reply:
Thanks!